### PR TITLE
Update rules engine

### PR DESCRIPTION
**Pull request by morganen**

### CODE REVIEW

File: content/md/first-steps/manage-your-rules.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 ee-only: true
 related: what-is-a-rule
 ---
 
 # Overview
 
-A rule allows you to enrich automatically some information in your products, according to automatic actions based on conditions. It applies on attributes, categories, system attributes (status, completeness...).  
-For more details, please refer to [What is a rule](what-is-a-rule.html).
+A rule allows you to automatically enrich some information in your products, according to automatic actions based on conditions. It applies on attributes, categories, system attributes (status, completeness...).  
+For more details, please refer to [the dedicated article](blabla.html).
```
Comment (Doodoune) [inline content/md/first-steps/manage-your-rules.md]: Does the page "blabla.html" exists?

Comment (morganen) [reply content/md/first-steps/manage-your-rules.md]: Nope, I'll add the link afterward. This is the second article that should be soon ready ;)

File: content/md/first-steps/manage-your-rules.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
-A rule allows you to enrich automatically some information in your products, according to automatic actions based on conditions. It applies on attributes, categories, system attributes (status, completeness...).  
-For more details, please refer to [What is a rule](what-is-a-rule.html).
+A rule allows you to automatically enrich some information in your products, according to automatic actions based on conditions. It applies on attributes, categories, system attributes (status, completeness...).  
+For more details, please refer to [the dedicated article](get-started-with-the-rules-engine.html).
 
 # View all rules
 
-The screen `Settings/Rules` displays all rules created.  
-You can search on the rule's code.
+The `Settings/Rules` screen displays all the rules that have been created and then imported in the PIM.
```
Comment (nmarniesse) [inline content/md/first-steps/manage-your-rules.md]: create and import rules are the same action (for now)

File: content/md/what-is/what-is-a-rule.md [hunk 3]
```diff
@@ -9,31 +9,31 @@ related: manage-your-rules
...
-- categorise new products
+- categorize new products
 - set a default value to an empty attribute
-- assign values, like a family, to new products
-- copy an attribute's value to another attribute (having the same attribute type)
+- assign values, such as a family, to new products
+- copy an attribute value to another attribute
 
-Rules can only be imported in Akeneo using a YML file, but you can manage them from the UI. Check [How to manage your rules](manage-your-rules.html) article. 
+Rules can only be created and then imported in Akeneo using a YML file, but you can manage them from the UI. Check our [How to manage your rules](manage-your-rules.html) article.
```
Comment (nmarniesse) [inline content/md/what-is/what-is-a-rule.md]: Same remark about the `create and then imported`

File: content/md/what-is/what-is-a-rule.md [hunk 4]
```diff
@@ -55,9 +55,10 @@ rules:
             -   type: set
                 field: camera_brand
                 value: canon_brand
-``` 
+```
+
+Now that you know what a rule is, let's dive into this powerful feature! Discover in [this article](.) all the conditions and actions available, and learn how to use them.
```
Comment (nmarniesse) [inline content/md/what-is/what-is-a-rule.md]: Missing link?

File: content/md/what-is/what-is-a-rule.md [hunk 5]
```diff
@@ -55,9 +55,10 @@ rules:
             -   type: set
                 field: camera_brand
                 value: canon_brand
-``` 
+```
```
Comment (mmetayer) [inline content/md/what-is/what-is-a-rule.md]: on line 50, I'm not sure we have to specify `.code` anymore, I guess 
```yaml
- field: camera_brand
  operator: 'NOT IN'
  value:
    - canon_brand
```
is enough

File: content/md/what-is/what-is-a-rule.md [hunk 6]
```diff
@@ -9,31 +9,31 @@ related: manage-your-rules
...
-- assign values, like a family, to new products
-- copy an attribute's value to another attribute (having the same attribute type)
+- assign values, such as a family, to new products
+- copy an attribute value to another attribute
 
-Rules can only be imported in Akeneo using a YML file, but you can manage them from the UI. Check [How to manage your rules](manage-your-rules.html) article. 
+Rules can only be created and then imported in Akeneo using a YML file, but you can manage them from the UI. Check our [How to manage your rules](manage-your-rules.html) article.
 
-Rules can be prioritised from 0 to 100, a rule with a priority of 100 will be applied before a rule having a priority of 10 or 50.
+Rules can be prioritized from 0 to 100: a rule with a priority of 100 will be applied before a rule having a priority of 10 or 50.
```
[the remark this answers is not in the document]
Comment (morganen) [reply content/md/what-is/what-is-a-rule.md]: I removed the number ;)

File: content/md/what-is/what-is-a-rule.md [hunk 7]
```diff
@@ -9,31 +9,31 @@ related: manage-your-rules
...
-A rule is a set of actions and conditions that allows you to automatise data enrichment. 
+A rule is a set of actions and conditions that allows you to automatize data enrichment.
 For instance, using rules you can automatically:
 - fill in attributes
-- categorise new products
+- categorize new products
 - set a default value to an empty attribute
-- assign values, like a family, to new products
-- copy an attribute's value to another attribute (having the same attribute type)
+- assign values, such as a family, to new products
```
[the remark this answers is not in the document]
Comment (morganen) [reply content/md/what-is/what-is-a-rule.md]: I removed "such as a family" ;)

File: content/md/first-steps/manage-your-rules.md [hunk 8]
```diff
@@ -65,13 +65,14 @@ You can also execute several rules at the same time:
 
 # Create a rule
 
-Rules are exclusively created by YML imports. You need to create your YML file like explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.
+Rules are exclusively created by YML imports. You need to create your YML file like explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.
```
Comment (camillebourjade) [inline content/md/first-steps/manage-your-rules.md]:
```suggestion
You can only create rules by YML  imports. You need to create your YML file as explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.  
```

File: content/md/first-steps/manage-your-rules.md [hunk 9]
```diff
@@ -65,13 +65,14 @@ You can also execute several rules at the same time:
 
 # Create a rule
 
-Rules are exclusively created by YML imports. You need to create your YML file like explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.
+Rules are exclusively created by YML imports. You need to create your YML file like explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.  
+We also detail in [this article](get-started-with-the-rules-engine.html), how to select products and the list of available actions.
```
Comment (camillebourjade) [inline content/md/first-steps/manage-your-rules.md]:
```suggestion
[Here](get-started-with-the-rules-engine.html) we also explain how to select products and the list of available actions.
```

File: content/md/first-steps/manage-your-rules.md [hunk 10]
```diff
@@ -65,13 +65,14 @@ You can also execute several rules at the same time:
...
 
 ::: info
 For more details about how to execute an import, please refer to [Import your data](imports.html).
```
Comment (camillebourjade) [inline content/md/first-steps/manage-your-rules.md]:
```suggestion
For more details on how to execute an import, please refer to [Import your data](imports.html).
```

File: content/md/first-steps/manage-your-rules.md [hunk 11]
```diff
@@ -65,13 +65,14 @@ You can also execute several rules at the same time:
...
 :::
 
-You can also export your rules. To export your rules, execute the `Rule export in YML` export job.
+You can also export your rules. For that, execute the `Rule export in YML` export job.
 
 ::: info
 For more details about how to execute an export, please refer to [Export your data](exports.html).
```
Comment (camillebourjade) [inline content/md/first-steps/manage-your-rules.md]:
```suggestion
For more details on how to execute an export, please refer to [Export your data](exports.html).
```

File: content/md/first-steps/manage-your-rules.md [hunk 12]
```diff
@@ -65,13 +65,14 @@ You can also execute several rules at the same time:
...
-Rules are exclusively created by YML imports. You need to create your YML file like explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.
+Rules are exclusively created by YML imports. You need to create your YML file like explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.  
+We also detail in [this article](get-started-with-the-rules-engine.html), how to select products and the list of available actions.
 
 ::: info
 For more details about how to execute an import, please refer to [Import your data](imports.html).
 :::
 
-You can also export your rules. To export your rules, execute the `Rule export in YML` export job.
+You can also export your rules. For that, execute the `Rule export in YML` export job.
```
Comment (camillebourjade) [inline content/md/first-steps/manage-your-rules.md]:
```suggestion
You can also export your rules. To do so, you need to execute the `Rule export in YML` export job.
```

File: content/md/first-steps/manage-your-rules.md [hunk 13]
```diff
@@ -99,6 +100,6 @@ If you do not see the `Delete` button, it might be because you do not have the r
 You can also delete several rules at the same time:
```
Comment (camillebourjade) [inline content/md/first-steps/manage-your-rules.md]:
```suggestion
You can also delete several rules at once:
```

File: content/md/first-steps/manage-your-rules.md [hunk 14]
```diff
@@ -99,6 +100,6 @@ If you do not see the `Delete` button, it might be because you do not have the r
...
 1.  Select one or several rules by ticking them on the left side of the screen  
 1.  Click on `Delete`
-1.  Then confirm the action in the pop in
+1.  Then confirm the action in the pop in.
```
Comment (camillebourjade) [inline content/md/first-steps/manage-your-rules.md]:
```suggestion
1.  Then, confirm the action in the pop in.
```

File: content/md/what-is/what-is-a-rule.md [hunk 15]
```diff
@@ -9,31 +9,31 @@ related: manage-your-rules
 
 # Definition of a rule
 
-A rule is a set of actions and conditions that allows you to automatise data enrichment. 
+A rule is a set of actions and conditions that allows you to automatize data enrichment.
 For instance, using rules you can automatically:
```
Comment (camillebourjade) [inline content/md/what-is/what-is-a-rule.md]:
```suggestion
For instance, rules allow you to automatically:
```

File: content/md/what-is/what-is-a-rule.md [hunk 16]
```diff
@@ -9,31 +9,31 @@ related: manage-your-rules
...
-- assign values, like a family, to new products
-- copy an attribute's value to another attribute (having the same attribute type)
+- assign values to new products
+- copy an attribute value to another attribute
 
-Rules can only be imported in Akeneo using a YML file, but you can manage them from the UI. Check [How to manage your rules](manage-your-rules.html) article. 
+Rules can only be created and imported in Akeneo using a YML file, but you can manage them from the UI. Check our [How to manage your rules](manage-your-rules.html) article.
 
-Rules can be prioritised from 0 to 100, a rule with a priority of 100 will be applied before a rule having a priority of 10 or 50.
+Rules can be prioritized: a rule with a priority of 100 will be applied before a rule having a priority of 10 or 50.
```
Comment (camillebourjade) [inline content/md/what-is/what-is-a-rule.md]:
```suggestion
Rules can be prioritized: a 100 priority rule will be applied before a 10 or 50 priority one.
```

Comment (camillebourjade) [reply content/md/what-is/what-is-a-rule.md]: Maybe explain the different priority levels just before

File: content/md/what-is/what-is-a-rule.md [hunk 17]
```diff
@@ -47,17 +47,18 @@ rules:
...
                 value:
                     - canon_brand
         actions:
             -   type: set
                 field: camera_brand
                 value: canon_brand
-``` 
+```
+
+Now that you know what a rule is, let's dive into this powerful feature! Discover in [this article](get-started-with-the-rules-engine.html) all the conditions and actions available, and learn how to use them.
```
Comment (camillebourjade) [inline content/md/what-is/what-is-a-rule.md]:
```suggestion
Now that you know what a rule is, let's dive into this powerful feature! Discover in [this article](get-started-with-the-rules-engine.html) all the available conditions and actions, and learn how to use them.
```

File: content/md/what-is/what-is-a-rule.md [hunk 18]
```diff
@@ -47,17 +47,18 @@ rules:
...
 
 ::: info
-Find more information about all the conditions/actions and create your own rules! To do so, refer to our technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure).
+You can refer to our [technical documentation](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) to find more information about the rules engine.
```
Comment (camillebourjade) [inline content/md/what-is/what-is-a-rule.md]:
```suggestion
You can refer to our [technical documentation](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) to find out more information about the rules engine.
```

**Pull request merged**